library/include/ck/library/host_tensor/device.hpp

```diff
@@ -1,13 +1,47 @@
 #ifndef DEVICE_HPP
 #define DEVICE_HPP
 
+#include "ck/options.hpp"
+
 #include <memory>
 #include <functional>
 #include <thread>
 #include <chrono>
 #include "hip/hip_runtime.h"
 #include "hip/hip_fp16.h"
 
+inline void hip_check(hipError_t x)
```

Comment:
It would be convenient to have a macro which would also report file and line where the error occur.

Reply:
Added